### PR TITLE
feat: add option to open link in new tab

### PR DESCRIPTION
**Pull request by jake-bassett**
## Description
This provides an option to open links in a new tab.

### CODE REVIEW

**Review by palbizu — APPROVED** [review]

File: projects/components/src/link/link.component.ts [hunk 1]
```diff
@@ -12,6 +12,8 @@ import { EMPTY, Observable } from 'rxjs';
     <a
       *htLetAsync="this.navData$ as navData"
       class="ht-link"
+      target="{{ this.openInNewTab ? '_blank' : '' }}"
```
Comment (aaron-steinfeld) [inline projects/components/src/link/link.component.ts:15]: how's this differ from the existing way of doing this?

This component takes an ExternalNavigationParams as an input, which includes window handling.

Comment (jake-bassett) [reply projects/components/src/link/link.component.ts:15]: Hmm. I thought it suspicious that we didn't have this already. I though that might be how, but it didn't seem to work when I was playing with it. I'll circle back and try again.

Comment (anandtiwary) [reply projects/components/src/link/link.component.ts:15]: Yep.

Comment (anandtiwary) [reply projects/components/src/link/link.component.ts:15]: ExternalNavigationParams option should support this.

**Pull request merged**